src/StorageSync/StorageSync/Properties/StorageSyncResources.resx

```diff
@@ -190,7 +190,7 @@
     <value>Unregister Storage Sync Server for Storage Sync Service '{0}' and all content in it</value>
   </data>
   <data name="ResetCertificateMessageFormat1" xml:space="preserve">
-    <value>Starting rollover certificate for ServerId: {serverId}</value>
+    <value>Starting rollover certificate for ServerId: {0}</value>
```

Comment:
So this was all about an exception from a verbose logging message?  Do we have any test coverage for this, or is this a scenario where test coverage is possible?

Reply:
hi Mark, test coverage by traditional means will not be possible since we would need a local COM call to be mocked. However, it is possible in general with additional effort of introducing appropriate mocking infra - I'd prefer to take this as a follow up item and filed a bug for that:
Bug 5460903: Test coverage is needed for Reset Certificate cmdlet in AZ

Meanwhile, I'd like to proceed with the fix as it is a high visibility for customers.